pkg/ery/cmd/init.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	toml "github.com/pelletier/go-toml"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/srvc/ery/pkg/app/command"
+	"github.com/srvc/ery/pkg/ery/di"
+	"go.uber.org/zap"
+)
+
+func newCmdInit(c di.AppComponent) *cobra.Command {
+	wd := c.Config().WorkingDir
+
+	var (
+		proj = filepath.Base(wd)
+		org  = filepath.Base(filepath.Dir(wd))
+		tld  = c.Config().TLD
+	)
+
+	hostname := strings.Join([]string{proj, org, tld}, ".")
+
+	cmd := &cobra.Command{
+		Use:   "init",
+		Short: "Genereate an ery's configuration file",
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
**[unparam]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
newCmdInit$1 - cmd is unused